### PR TITLE
[Dependent] Cypress test. Project functionality.

### PR DESCRIPTION
**Pull request by dvkruchinin**
### Motivation and context
Depend on #2255 
Cypress test to check base actions on project.
Functionality for creating a project. Some necessary checks after executing the cypress command are moved to the body of the cypress command.

### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [x] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2020 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

**Review by ActiveChooN — COMMENTED** [review]
Great work!
Have another one test case to add.
1. Assign second task created in the project to admin user
2. Login with second user
3. Go to tasks page, there is only should be first task from project

**Comment by dvkruchinin** [comment]
@ActiveChooN 
> Have another one test case to add.
> 
> 1. Assign second task created in the project to admin user
> 2. Login with second user
> 3. Go to tasks page, there is only should be first task from project

Test case added. Also I move cypress command for projects functionality to different command file (```commands_projects.js```). I think it`s would be better for readability.

File: tests/cypress/integration/remove_users_tasks_projects.js [hunk 1]
```diff
@@ -61,4 +61,24 @@ describe('Delete users and tasks created during the test run.', () => {
             }
         });
     });
+    it('Get a list of projects and delete them all', () => {
+        cy.request({
+            url: '/api/v1/projects?page_size=1000',
```
[the remark this answers is not in the document]
Comment (dvkruchinin) [reply tests/cypress/integration/remove_users_tasks_projects.js]: Thanks. I applied this in a new commit

**Comment by ActiveChooN** [comment]
LGTM,

@bsekachev, could you please merge PR?

**Review by ActiveChooN — APPROVED** [review]

**Pull request merged**